pycfmodel/model/resources/iam_group.py

```diff
@@ -35,4 +36,14 @@ class IAMGroup(Resource):
 
     TYPE_VALUE: ClassVar = "AWS::IAM::Group"
     Type: str = TYPE_VALUE
-    Properties: Resolvable[IAMGroupProperties]
+    Properties: Optional[Resolvable[IAMGroupProperties]]
+
+    @property
+    def policy_documents(self) -> List[OptionallyNamedPolicyDocument]:
+        result = []
+        if self.Properties.Policies:
```

Comment:
Is this needed? Just asking as the `iam_role` resource file does not include this check and goes straight to the `for` loop.

Reply:
Policies is optional and it can be None

Reply:
Cool - in that case shall we add this check to the `iam_role` resource? https://github.com/Skyscanner/pycfmodel/pull/51/files#diff-cdbcad10dd72e985485f15bd8ecfe4155a8e82923e12e8a829bb8c56d1b1ae34R50

Reply:
Good catch, I've fixed it.